frontend/app/components/nav.jsx

```diff
@@ -8,17 +9,21 @@ class Navigation extends React.Component {
 
   render() {
     return (
-      <Navbar inverse fixedTop >
+      <Navbar inverse fixedTop collapseOnSelect>
         <Navbar.Header>
           <Navbar.Brand>
-            <a href="#">Ickly</a>
+            <a href="/">Ickly</a>
```

Comment:
This seems to trigger a hard page refresh for some reason. Maybe because of the `exact path` directive above? It's hard to say but it would be much preferable for this to simply switch to the `Home` view instead of refreshing the page.

I will still approve this since I don't want this to block progress, but I'll leave it up to you whether you want to fix it before moving forward.

Reply:
Ok, i'll look into it.